src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -58,33 +53,114 @@ If you'll need to do a good amount of partitioning of users' access to accounts
+## Step 2: Set up organization [#page2]
 
+![User migration page 2](./images/user-migration-page-2.png "user-migration-page-2.png")
+
+You may choose a) a guided setup that allows more configuration options, or b) an automatic setup with fewer steps. Some notes on the automatic setup:
+
+* It won't transition SSO or [AUM-implemented SCIM configurations](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Instead, users will be provisioned manually and will log in manually with their username and password. 
+* It provides all users with access to all accounts. If you use SSO or AUM today or require some accounts to be inaccessible by certain user groups, don't use the automatic option.
+
+## Step 3: Name your organization [#page3]
+
+![User migration page 3](./images/user-migration-page-3.png "user-migration-page-3.png")
+
+Name your organization something descriptive and easily recognizable.
+
+## Step 4: Authentication domain settings [#page4]
+
+![User migration page 4](./images/user-migration-page-4.png "user-migration-page-4.png")
+
+This section controls how users are provisioned (added to New Relic) and how they authenticate (log in). The two sections are described in more detail: 
+
+### Managing users [#managing-users]
+
+For managing users, you can select **Manually** or **Identity provider**. 
+
+The option to use your identity provider (like Okta, OneLogin, and Azure AD) to provision users is available only if your organization has [Enterprise edition](https://newrelic.com/pricing). If you choose **Identity provider**, you must follow [additional steps for using specific identity providers](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Once you do that, these users are created on the new user model and synced in New Relic based on your identity provider configuration. After you complete this, you can confirm that these users have been added by seeing if they’re in the new [**User management** UI](#manage-users).
+
+If you’ve selected the guided workflow, we’ll match these users by email address. There will be no need to later complete the user import step for these users given they’ll have been brought into New Relic via your identity provider. You can continue the migration workflow and skip ahead to the **Migrate assets** step.
+
+### Login methods [#login-methods]
+
+If you chose **Manually** for [managing users](#managing-users), you have a choice for how those users log in. You can select either a) email/password login or b) single sign on (SSO). Note that SSO is available only for organizations with Pro or Enterprise edition.
```

Comment:
available only for organizations with a Pro or Enterprise edition? The Pro or Enterprise edition?

Reply:
i think 'with Pro or Enterprise edition' makes most sense to me but let me know if you think this seems clearly wrong